Tavisca.Bootcamp.LanguageBasics.Exercise1/FixMultiplication.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+
+namespace Tavisca.Bootcamp.LanguageBasics.Exercise1
+{
+    class FixMultiplication
+    {
+        private static float value1, value2, answer;
+        private static string temp1 = null, temp2 = null;
+
+        public static int FindDigit(string equation)
+        {
+            string[] splitOnEquals = equation.Split('=');
+            string stringAnswer = splitOnEquals[1];
+            string[] splitOnMultiply = splitOnEquals[0].Split('*');
+            string stringValue1 = splitOnMultiply[0];
+            string stringValue2 = splitOnMultiply[1];
+
+            if (stringAnswer.Contains('?'))
+            {
+                CalculateCorrectAnswer(stringValue1, stringValue2, stringAnswer);
+            }
+            else if (stringValue1.Contains('?'))
+            {
+                CalculateCorrectValue1(stringValue1, stringValue2, stringAnswer);
+            }
+            else if (stringValue2.Contains('?'))
+            {
+                CalculateCorrectValue2(stringValue1, stringValue2, stringAnswer);
+            }
+
+            return FindMissingDigit();
+        }
+
+        private static void CalculateCorrectAnswer(string stringValue1, string stringValue2, string stringAnswer)
+        {
+            value1 = float.Parse(stringValue1);
+            value2 = float.Parse(stringValue2);
+            answer = value1 * value2;
+            temp1 = answer.ToString();
+            temp2 = stringAnswer;
+        }
+
+        private static void CalculateCorrectValue1(string stringValue1, string stringValue2, string stringAnswer)
+        {
+            value2 = float.Parse(stringValue2);
+            answer = float.Parse(stringAnswer);
```

Comment:
It seems scope of variables value1 value2 and answer is limited to these methods..
So you can declare these variables as local to a function..